src/Serialization/GAP.jl

```diff
@@ -282,18 +276,17 @@ install_GAP_deserialization(:IsPcGroup,
             length(relord))
       fam = GAPWrap.ElementsFamily(GAPWrap.FamilyObj(F))
       rws = GAP.Globals.SingleCollector(F, GapObj(relord))::GapObj
-      reli = load_object(s, Vector, d[:power_reli], Int)
-      rels = load_object(s, Vector, d[:power_rels], (Vector{Int}, Int))
-      for k in 1:length(reli)
-        GAP.Globals.SetPower(rws, reli[k],
-          GapObj(GAPWrap.ObjByExtRep(fam, GapObj(rels[k]))))
+      for data_rel in d[:power_rels]
+        (i, elm) = load_object(s, Tuple{Int, Vector{Int}}, data_rel,
```

Comment:
I am not sure if this work?
```suggestion
        (i, elm) = load_object(s, Tuple, data_rel, [Int, Vector{Int}])```

in any case you can certainly just replace Tuple{Int, Vector{Int}} with Tuple which should help a bit